tasks.py

```diff
@@ -248,8 +260,36 @@ def deploy_core(ctx, name, version=None, no_tests=False):
 
 
 @task
-def deploy_lambda_package(ctx, name):
-    aws_lambda.deploy(os.getcwd(), local_package='../..', requirements='../../requirements.txt')
+def deploy_lambda_package(ctx, name, suffix):
+    # create the temporary local dev lambda directories
+    if suffix:
+        new_name = name + '_' + suffix
+        new_src = '../' + new_name
+        cmd_mkdir = "rm -fr %s; mkdir -p %s" % (new_src, new_src)
+        cmd_copy = "cp -r * %s" % new_src
+        cmd_cd = "cd %s" % new_src
+        cmd_modify_cfg = "sed 's/%s/%s/g' config.yaml > config.yaml#" % (name, new_name)
+        cmd_replace_cfg = "mv config.yaml# config.yaml"
+        cmd = ';'.join([cmd_mkdir, cmd_copy, cmd_cd, cmd_modify_cfg, cmd_replace_cfg])
+        print(cmd)
+        run(cmd)
+    else:
+        new_name = name
+    with chdir(new_src):
+        aws_lambda.deploy(os.getcwd(), local_package='../..', requirements='../../requirements.txt')
+    # add roles and environment variables
+    role_arn = 'arn:aws:iam::643366669028:role/lambda_full_s3'  # temporarily hardcorded
```

Comment:
So the arn is in the config.yaml and will be set via aws_lambda.deploy so don't think you need to set it a second time.

Reply:
just need to make sure it's in the config.yml that is being copied across.

Reply:
Good point!

Reply:
I removed the part that adds roles